instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py

```diff
@@ -167,4 +202,14 @@ def process_response(self, request, response):
             detach(request.environ.get(self._environ_token))
             request.META.pop(self._environ_token)
 
+        try:
+            metric_recorder = getattr(settings, "OTEL_METRIC_RECORDER", None)
+            if metric_recorder:
```

Comment:
It think it is better to be specific in the check here, since several things (like an empty string) also evaluate to `False`.

```suggestion
            if metric_recorder is not None:
```